client/gutenberg/extensions/prev-next/block.js

```diff
@@ -31,22 +31,16 @@ const blockAttributes = {
 const save = ( { attributes: { prev, next }, className, isEditor } ) =>
 	prev || next ? (
 		<div className={ isEditor ? className : '' }>
-			{ prev ? <a href={ prev }>← Prev</a> : <span> </span> }
-			{ next ? <a href={ next }>Next →</a> : <span> </span> }
+			{ prev ? <a href={ prev }>← { __( 'Prev' ) }</a> : <span> </span> }
+			{ next ? <a href={ next }>{ __( 'Next' ) } →</a> : <span> </span> }
```

Comment:
Does this go the correct direction for rtl?

🤯

Reply:
Good question! It doesn't. :-) This block's CSS wasn't RTL friendly anyway (fixed in https://github.com/Automattic/wp-calypso/pull/27181/commits/5e6805a59248d2b8811ab536fce7fc70b1f71681) but while at it, I realised that changing this markup would "deprecate" any existing blocks added to posts so I'm backing from up making any changes to this before I learn more about [deprecating block markup](https://wordpress.org/gutenberg/handbook/block-api/deprecated-blocks/). This block will be a great simple study-case later on.

So it'll remain broken for RTL readers.